source/administration/changelog.md

```diff
@@ -127,7 +127,30 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
  - Added ``lodash``, ``memoize-one``, and ``sass`` to https://github.com/mattermost/mattermost-webapp.
  - Added ``react-native-startup-time`` and removed ``react-native-status-bar-size`` from https://github.com/mattermost/mattermost-mobile.
 
+### Upcoming Deprecations in Mattermost v6.0
+
+The following deprecations are planned for the Mattermost v6.0 release, which is scheduled for 2021/09/15. This list is subject to change prior to the release.
+
+1. [Legacy Command Line Tools](https://docs.mattermost.com/administration/command-line-tools.html). All commands have been fully replaced by [MMCTL](https://docs.mattermost.com/administration/mmctl-cli-tool.html) and new commands have been added over the last few months, making this tool a full and robust replacement. 
+
+2. [Slack Import via the Web App](https://docs.mattermost.com/administration/migrating.html?highlight=mmetl#migrating-from-slack-using-the-mattermost-web-app). The Slack import tool accessible via the Team Setting menu is being replaced by the [MMETL](https://docs.mattermost.com/administration/migrating.html?highlight=mmetl#migrating-from-slack-using-the-mattermost-mmetl-tool-and-bulk-import) tool that is much more comprehensive for types of data it can assist in uploading. 
+
+3. MySQL versions below to 5.7.7. Minimum support will now be for 5.7.8. This version introduced a native JSON data type that lets us improve performance and scalability of several database fields (most notably Users and Posts props), Additionally, 5.6 (our current minimum version) reached [EOL in February 2021](https://www.mysql.com/support/eol-notice.html).
+
+4. Elasticsearch 5 & 6 - [versions 5.x reached EOL in March of 2019, and versions 6.x reached EOL in November 2020](https://www.elastic.co/support/eol).
+
+5. Windows 7 reached [EOL in January 2020](https://support.microsoft.com/en-us/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962). We will no longer provide support for the Desktop App issues on Windows 7.
+
+6. [DisablePostMetaData](https://docs.mattermost.com/administration/config-settings.html#disable-post-metadata) configuration setting.
+
+7. [Experimental Timezone](https://docs.mattermost.com/administration/config-settings.html#timezone) configuration setting.
+
```

Comment:
I think we're missing a lot of configs here: https://focalboard-community.octo.mattermost.com/workspace/suciie6fxpbz9rryiwuxqa8yqh?id=0bdf3bf4-dc53-4274-88a6-121003f431e4&v=8763e825-c543-4ec5-809a-73fa705dc2a8&c=247af414-fcf9-4c6b-857e-564fe49c16d3

Reply:
@esethna The list comes from the official communications from PMs https://docs.google.com/document/d/13XseTWS2Bp7hsmiX-6AqMkiFMRwQOqMduNPpUizjdR8/edit. @wiersgallak can confirm.

Reply:
The "legacy sidebar" items listed in that doc are all these: 
![image](https://user-images.githubusercontent.com/14333569/122998263-ae084c80-d361-11eb-9060-5ed70673023f.png)